src/Sylius/Component/Core/OrderProcessing/OrderPricesRecalculator.php

```diff
@@ -44,8 +46,20 @@ public function process(BaseOrderInterface $order): void
                 continue;
             }
 
+            /** @var ProductVariantInterface $variant */
+            $variant = $item->getVariant();
+
+            /** @var ChannelPricingInterface $channelPricing */
+            $channelPricing = $variant->getChannelPricingForChannel($channel);
+
+            $originalPrice = $channelPricing->getOriginalPrice();
+
+            if ($originalPrice !== null) {
+                $item->setOriginalPrice($channelPricing->getOriginalPrice());
+            }
```

Comment:
By default this value is also null, so this `if` is not needed IMO:
```suggestion
            $item->setOriginalPrice($originalPrice->getOriginalPrice());
```